pystac/collection.py

```diff
@@ -37,6 +37,8 @@
     from pystac.provider import Provider as Provider_Type
 
 T = TypeVar("T")
+TemporalIntervals = Union[List[List[datetime]], List[List[Optional[datetime]]]]
+TemporalIntervalsLike = Union[TemporalIntervals, List[datetime], List[List[datetime]]]
```

Comment:
I'm probably just not understanding the typing property, but this looks like `List[List[datetime]]` is duplicated by this union?

Reply:
Oh shoot, you're right. I meant that to be:

```python
TemporalIntervalsLike = Union[TemporalIntervals, List[datetime], List[Optional[datetime]]]
```

Reply:
Fixed in the latest commit.